### ISSUE TITLE
报错torch.cat(): expected a non-empty list of Tensors

### ISSUE THREAD

**Issue by jjxyhb** [HZAI-ZJNU/Mamba-YOLO#53]
train: Scanning /media/dell/DATA/coco2017/train2017.cache... 0 images, 118287 backgrounds, 0 corrupt: 100%|██████████| 118287/118287 [00:00<?, ?it/s]   
WARNING ⚠️ No labels found in /media/dell/DATA/coco2017/train2017.cache, training may not work correctly. See https://docs.ultralytics.com/datasets for d
ataset formatting guidance.                                                                                                                             
val: Scanning /media/dell/DATA/coco2017/val2017.cache... 0 images, 5000 backgrounds, 0 corrupt: 100%|██████████| 5000/5000 [00:00<?, ?it/s]             
WARNING ⚠️ No labels found in /media/dell/DATA/coco2017/val2017.cache, training may not work correctly. See https://docs.ultralytics.com/datasets for dat
aset formatting guidance.                                                                                                                               
Plotting labels to /home/dell/CV408/hb/Mamba-YOLO/output_dir/mscoco/mambayolo_n5/labels.jpg...                                                          
zero-size array to reduction operation maximum which has no identity

**Comment by yuxr20** [comment]
请问您问题解决了吗，我这遇到了完全相同的报错
